packages/@aws-cdk/aws-applicationautoscaling/test/test.cron.ts

```diff
@@ -26,4 +26,22 @@ export = {
     }, /Duration cannot be 0/);
     test.done();
   },
+
+  'rate can be token'(test: Test) {
+    const stack = new Stack();
+    const invokeScheduleInMinutes = new CfnParameter(stack, 'invokeScheduleInMinutes', {
+      type: 'Number',
+      description: 'The minutes between scheduled invocations.',
+      default: 5,
+    });
+    test.equal('rate(<token> minutes)', appscaling.Schedule.rate(Duration.minutes(invokeScheduleInMinutes.valueAsNumber))
+      .expressionString);
+    test.done();
+  },
+
+  'rate can be in minutes'(test: Test) {
```

Comment:
curious: why do we need this extra test case?

Reply:
I wanted to make sure we had something around regular minutes.